VMBackup/main/fsfreezer.py

```diff
@@ -183,15 +184,15 @@ def freeze_safe(self,timeout):
                 try:
                     if not os.path.isdir('/etc/azure/MicrosoftRecoverySvcsSafeFreezeLock'):
                         os.mkdir('/etc/azure/MicrosoftRecoverySvcsSafeFreezeLock')
-                    file = open("/etc/azure/MicrosoftRecoverySvcsSafeFreezeLock/SafeFreezeLockFile","w")
+                    self.file = open("/etc/azure/MicrosoftRecoverySvcsSafeFreezeLock/SafeFreezeLockFile","w")
```

Comment:
do we need to keep the file pointer open till thaw.
can we close here and open in thaw at the time of releasing the lock ?

Reply:
No. If we close the file then another process is able to acquire the lock on it.